data/json/monsters/rodentkin.json

```diff
@@ -34,8 +34,8 @@
     "melee_dice_sides": 1,
     "melee_damage": [ { "damage_type": "cut", "amount": 1 } ],
     "dodge": 2,
-    "vision_day": 30,
-    "vision_night": 15,
+    "vision_day": 15,
+    "vision_night": 10,
```

Comment:
Why's this guy got better vision than the other albino lab rat?

Reply:
The buggers are mutant lil buggers, felt it made sense if they could see a bit better

Reply:
> The buggers are mutant lil buggers, felt it made sense if they could see a bit better

Nah they're regular rats high on purifier, made specifically to be exact replicas that are able to upgrade.

Reply:
Alright, I'll change it then